x-pack/legacy/plugins/siem/public/components/embeddables/embedded_map.tsx

```diff
@@ -4,41 +4,74 @@
+import { SavedObjectFinder } from 'ui/saved_objects/components/saved_object_finder';
 
-import styled from 'styled-components';
-import { start } from '../../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public/legacy';
 import { EmbeddablePanel } from '../../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public';
-
+import { start } from '../../../../../../../src/legacy/core_plugins/embeddable_api/public/np_ready/public/legacy';
+import { DEFAULT_INDEX_KEY } from '../../../common/constants';
 import { getIndexPatternTitleIdMapping } from '../../hooks/api/helpers';
 import { useIndexPatterns } from '../../hooks/use_index_patterns';
-import { useKibanaUiSetting } from '../../lib/settings/use_kibana_ui_setting';
-import { DEFAULT_INDEX_KEY } from '../../../common/constants';
-import { useKibanaPlugins } from '../../lib/compose/kibana_plugins';
 import { useKibanaCore } from '../../lib/compose/kibana_core';
-import { useStateToaster } from '../toasters';
+import { useKibanaPlugins } from '../../lib/compose/kibana_plugins';
+import { useKibanaUiSetting } from '../../lib/settings/use_kibana_ui_setting';
 import { Loader } from '../loader';
+import { useStateToaster } from '../toasters';
+import { Embeddable } from './embeddable';
+import { EmbeddableHeader } from './embeddable_header';
+import { createEmbeddable, displayErrorToast, setupEmbeddablesAPI } from './embedded_map_helpers';
 import { IndexPatternsMissingPrompt } from './index_patterns_missing_prompt';
-import { MapEmbeddable, SetQuery } from './types';
+import { MapToolTip } from './map_tool_tip/map_tool_tip';
 import * as i18n from './translations';
+import { MapEmbeddable, SetQuery } from './types';
 
-import { createEmbeddable, displayErrorToast, setupEmbeddablesAPI } from './embedded_map_helpers';
-import { MapToolTip } from './map_tool_tip/map_tool_tip';
+interface EmbeddableMapProps {
+  maintainRatio?: boolean;
+}
 
-const EmbeddableWrapper = styled(EuiFlexGroup)`
-  position: relative;
-  height: 400px;
-  margin: 0;
+const EmbeddableMap = styled.div.attrs({
+  className: 'siemEmbeddable__map',
+})<EmbeddableMapProps>`
+  ${({ maintainRatio, theme }) => css`
+    .euiPanel {
+      border: none;
+      box-shadow: none;
+    }
+
+    .mapToolbarOverlay__button {
+      display: none;
+    }
+
+    ${maintainRatio &&
+      css`
+        padding-top: calc(3 / 4 * 100%); //4:3 (standard) ratio
+        position: relative;
+
+        @media only screen and (min-width: ${theme.eui.euiBreakpoints.m}) {
+          padding-top: calc(9 / 32 * 100%); //32:9 (ultra widescreen) ratio
+        }
 
-  .mapToolbarOverlay__button {
-    display: none;
-  }
+        @media only screen and (min-width: 1441px) and (min-height: 901px) {
+          padding-top: calc(9 / 21 * 100%); //21:9 (ultrawide) ratio
+        }
+
+        .euiPanel {
+          bottom: 0;
+          left: 0;
+          position: absolute;
+          right: 0;
+          top: 0;
```

Comment:
I think some styles are sneaking their way down into the LayerToC, as it goes full-map when expanded now:

<img width=500 src="https://user-images.githubusercontent.com/2946766/67590441-4c20bb00-f718-11e9-93ec-0ab3471f6a7c.png">

Reply:
Ah, good catch. I didn't test the inner map menus. Will correct now. Thanks!